src/lib/Components/Inbox/Bids/index.tsx

```diff
@@ -8,8 +8,21 @@ import ActiveBid from "./ActiveBid"
 
 const Container = styled.View`margin: 20px 0 40px;`
 
-class ActiveBids extends React.Component<any, any> {
+interface Props extends RelayProps {
+  onDataLoaded?: (hasData: boolean) => void
```

Comment:
this callback allows the parent component (Inbox) to know whether there's data or not.
since relay only exposes the data in this component